v2/devices/dandelion.yml

```diff
@@ -85,7 +85,7 @@ operating_systems:
                 - url: "https://github.com/ubuntu-touch-mt6765/ut-images/raw/main/recovery_dandelion.img"
```

Comment:
You could also fix this to a commit, like `6c6debdd191c4837595b8249581c7351f355fada` instead of `main` to be able to update the main branch but still not break the installer.

Reply:
But fixing it to a commit would make it only fetch the recovery image from that point, no?

Reply:
Or better use GitHub releases feature as that uses CDN and is meant for people downloading individual files, as opposed to extracting a file from .git repo on request.

Reply:
@muhammad23012009 that's expected, as you need to update config for a new one or checksums will not match.